db/sqldb/initiative_user.go

```diff
@@ -0,0 +1,110 @@
+package sqldb
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/RMI/pacta/db"
+	"github.com/RMI/pacta/pacta"
+	"github.com/jackc/pgx/v4"
+)
+
+const initiativeUserRelationshipSelectColumns = `initiative_id, user_id, member, manager, updated_at`
+
+func (d *DB) InitiativeUserRelationship(tx db.Tx, iid pacta.InitiativeID, uid pacta.UserID) (*pacta.InitiativeUserRelationship, error) {
+	rows, err := d.query(tx, `
+		SELECT `+initiativeUserRelationshipSelectColumns+`
+		FROM initiative_user_relationship 
+		WHERE initiative_id = $1 AND user_id = $2;`, iid, uid)
+	if err != nil {
+		return nil, fmt.Errorf("querying initiative_user_relationship: %w", err)
+	}
+	iurs, err := rowsToInitiativeUserRelationships(rows)
+	if err != nil {
+		return nil, fmt.Errorf("translating rows to initiative_user_relationships: %w", err)
+	}
+	return exactlyOne("initiative_user_relationship", fmt.Sprintf("%s:%s", iid, uid), iurs)
+}
+
+func (d *DB) InitiativeUserRelationshipsByUser(tx db.Tx, uid pacta.UserID) ([]*pacta.InitiativeUserRelationship, error) {
+	rows, err := d.query(tx, `
+		SELECT `+initiativeUserRelationshipSelectColumns+`
+		FROM initiative_user_relationship 
+		WHERE user_id = $1;`, uid)
+	if err != nil {
+		return nil, fmt.Errorf("querying initiative_user_relationship: %w", err)
+	}
+	iurs, err := rowsToInitiativeUserRelationships(rows)
+	if err != nil {
+		return nil, fmt.Errorf("translating rows to initiative_user_relationships: %w", err)
+	}
+	return iurs, nil
+}
+
+func (d *DB) InitiativeUserRelationshipsByInitiatives(tx db.Tx, iid pacta.InitiativeID) ([]*pacta.InitiativeUserRelationship, error) {
+	rows, err := d.query(tx, `
+		SELECT `+initiativeUserRelationshipSelectColumns+`
+		FROM initiative_user_relationship 
+		WHERE initiative_id = $1;`, iid)
+	if err != nil {
+		return nil, fmt.Errorf("querying initiative_user_relationship: %w", err)
+	}
+	iurs, err := rowsToInitiativeUserRelationships(rows)
+	if err != nil {
+		return nil, fmt.Errorf("translating rows to initiative_user_relationships: %w", err)
+	}
+	return iurs, nil
+}
+
+func (d *DB) PutInitiativeUserRelationship(tx db.Tx, iur *pacta.InitiativeUserRelationship) error {
+	if err := validateInitiativeUserRelationshipForPut(iur); err != nil {
+		return fmt.Errorf("validating initiative_user_relationship for put: %w", err)
+	}
+	updatedAt := time.Now()
```

Comment:
Similar note here about time, I'd just inline this in the SQL or at least have a mock `d.now()`